src/plugins/event-plugins/XhrPlugin.ts

```diff
@@ -95,13 +95,13 @@ export const XHR_PLUGIN_ID = 'xhr';
  */
 export class XhrPlugin extends MonkeyPatched<XMLHttpRequest, 'send' | 'open'> {
     private config: HttpPluginConfig;
-    private xhrMap: Map<XMLHttpRequest, XhrDetails>;
+    private map: Map<XMLHttpRequest, XhrDetails>;
```

Comment:
Why are we doing all this renaming? This makes it difficult to understand the change. For example, if someone is debugging a related issue in the future. 

My understanding was that this would be reverted before being merged.